packages/firestore/scripts/run-tests.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+var __assign =
```

Comment:
Is it possible to mark this file as generated code? I started writing a whole bunch of comments on this and then only realized later that you hadn't written this.

Reply:
I changed the build rule to run these through `terser`, which should make this more obvious.